downstream/modules/aap-hardening/con-logging-log-capture.adoc

```diff
@@ -7,8 +7,8 @@
+This guide recommends that you configure {PlatformNameShort} and the underlying RHEL systems to collect logging and auditing centrally, rather than reviewing it on the local system. {ControllerNameStart} must be configured to use external logging to compile log records from multiple components within the controller server. The events occurring must be time-correlated to conduct accurate forensic analysis. This means that the controller server must be configured with an NTP server that is also used by the logging aggregator service, as well as the targets of the controller. The correlation must meet certain industry tolerance requirements. In other words, there might be a varying requirement that time stamps of different logged events must not differ by any amount greater than X seconds. This capability should be available in the external logging service.
 
-Another critical capability of logging is the ability to use cryptography to protect the integrity of log tools. Log data includes all information (e.g., log records, log settings, and log reports) needed to successfully log information system activity. It is common for attackers to replace the log tools or inject code into the existing tools for the purpose of providing the capability to hide or erase system activity from the logs. To address this risk, log tools must be cryptographically signed to provide the capability to identify when the log tools have been modified, manipulated, or replaced. An example is a checksum hash of the file or files. {ControllerNameStart} server log tools must use cryptographic mechanisms to protect the integrity of the tools or allow cryptographic protection mechanisms to be applied to their tools.
+Another critical capability of logging is the ability to use cryptography to protect the integrity of log tools. Log data includes all information (for example, log records, log settings, and log reports) needed to successfully log information system activity. It is common for attackers to replace the log tools or inject code into the existing tools to hide or erase system activity from the logs. To address this risk, log tools must be cryptographically signed so that you can identify when the log tools have been modified, manipulated, or replaced. For example, one way to validate that the log tool(s) have not been modified, manipulated or replaced is to use a checksum hash against the tool file(s). This ensures the integrity of the  tool(s) have not been compromised.
```

Comment:
Suggest rewriting "This ensures the integrity of the tool(s) **has** not been compromised." It refers to the integrity (singular) and not tools (plural).

Reply:
Done